### ISSUE TITLE
Function not formatted correctly

### ISSUE THREAD

**Issue by GISED-Link** [Barteling/STweep#67]
**Describe the bug**
This code does not format correctly:
```pascal
IF (STATE_READY = state) AND (LEN(mode_to_string) = 2) THEN
Tc2_Utilities.CONCAT2(ADR('SM'), ADR(mode_to_string), ADR(command_param_data), SIZEOF(command_param_data));
END_IF
```

but this one does:
```
IF (STATE_READY = state) AND (LEN(mode_to_string) = 2) THEN
Tc2_Utilities.CONCAT2(ADR(yolo), ADR(mode_to_string), ADR(command_param_data), SIZEOF(command_param_data));
END_IF
```

**Expected behavior**
```pascal
IF (STATE_READY = state) AND (LEN(mode_to_string) = 2) THEN
    Tc2_Utilities.CONCAT2(ADR('SM'), ADR(mode_to_string), ADR(command_param_data), SIZEOF(command_param_data));
END_IF
```

**STweep version info**
STWeep for TwinCAT 4.0.4

**IDE version info**
I use Twincat shell

**Additional context**
It is the ADR('some nice strings) that STWeep does not like

**Comment by Barteling** [comment]
Got it, thanks for the thorough research!  Stweeps parser does indeed not expect a string literal in a ADR function.

It will be fixed in the next release

**Comment by Barteling** [comment]
Fixed in 4.1.0- Beta
https://www.stweep.com/beta-releases/
